backend/executor.go

```diff
@@ -311,7 +311,7 @@ func (g *grpcExecutor) StartInstance(ctx context.Context, req *protos.CreateInst
 	defer span.End()
 
 	e := helpers.NewExecutionStartedEvent(req.Name, instanceID, req.Input, nil, helpers.TraceContextFromSpan(span))
-	if err := g.backend.CreateOrchestrationInstance(ctx, e); err != nil {
+	if err := g.backend.CreateOrchestrationInstance(ctx, e, WithOrchestrationIdReusePolicy(req.OrchestrationIdReusePolicy)); err != nil {
```

Comment:
Default value for `req.OrchestrationIdReusePolicy` will be
```go
policy := &protos.OrchestrationIdReusePolicy{
        // empty enum in grpc will have the first value as default
	Action: protos.CreateOrchestrationAction_ERROR,
        // repeated message will have default value as empty slice
	OperationStatus: []protos.OrchestrationStatus{},
}
```
So it will have the same behavior as before.